### DISCUSSION TITLE
📣 POLL: Are you currently using SP-API notifications?

### DISCUSSION THREAD

**Discussion by rodrifed** [amzn/selling-partner-api-samples#39] [Polls]
The SP-API team is looking for your feedback in order to identify the resources that would help you the most in your API integration journey.
Please answer the poll below, raise your voice, and help us shape the future of SP-API content!

**Comment by johnkw** [comment]
We do send feeds, but they complete within a minute or so generally, so setting up a server just to receive that seems like overkill.
